components/combobox/index.css

```diff
@@ -0,0 +1,609 @@
+  /* Hide native outline */
+  outline-color: transparent;
+
+  &.is-focused,
+  &:focus-within {
+    .spectrum-Combobox-input {
+      background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-focus, var(--spectrum-combobox-background-color-focus)));
+      border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-focus, var(--spectrum-combobox-border-color-focus)));
+      color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-focus, var(--spectrum-combobox-font-color-focus)));
+    }
+  }
+
+  &.is-keyboardFocused {
+    .spectrum-Combobox-input {
+      background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-key-focus, var(--spectrum-combobox-background-color-key-focus)));
+      border-color: var(--highcontrast-combobox-border-color, var(--mod-combobox-border-color-key-focus, var(--spectrum-combobox-border-color-key-focus)));
+      color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-key-focus, var(--spectrum-combobox-font-color-key-focus)));
+    }
+  }
+
+  &:hover {
+    .spectrum-Combobox-input {
+      background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-hover, var(--spectrum-combobox-background-color-hover)));
+      border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-hover, var(--spectrum-combobox-border-color-hover)));
+      color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-hover, var(--spectrum-combobox-font-color-hover)));
+    }
+  }
+
+  &:focus:hover,
+  &.is-focused:hover,
+  &:focus-within:hover {
+    .spectrum-Combobox-input {
+      background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-focus-hover, var(--spectrum-combobox-background-color-focus-hover)));
+      border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-focus-hover, var(--spectrum-combobox-border-color-focus-hover)));
+      color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-focus-hover, var(--spectrum-combobox-font-color-focus-hover)));
+    }
+  }
+
+  .spectrum-Combobox &.is-invalid,
+  .spectrum-Combobox &.is-loading {
+    .spectrum-Combobox-input {
+      padding-inline-end: calc(
+        var(--mod-combobox-button-width, var(--spectrum-combobox-button-width)) +
+        var(--mod-combobox-spacing-inline-icon-to-button, var(--spectrum-combobox-spacing-inline-icon-to-button)) +
+        var(--mod-combobox-icon-size, var(--spectrum-combobox-icon-size)) +
+        var(--mod-spectrum-combobox-spacing-inline-end-edge-to-text, var(--spectrum-combobox-spacing-inline-end-edge-to-text)) -
+        var(--mod-combobox-button-inline-offset, var(--spectrum-combobox-button-inline-offset, 0px))
+      );
+    }
+  }
+
+  .spectrum-Combobox &.is-invalid {
```

Comment:
Same question as above. Does this CSS get reached?

Reply:
(note: same as [comment above](https://github.com/adobe/spectrum-css/pull/1556#discussion_r1142545517))